lib/netpacket/netpacket_read.c

```diff
@@ -12,9 +12,9 @@
 
 /**
  * netpacket_read_file(NPC, machinenum, class, name, size, callback):
- * Construct and send a NETPACKET_READ_FILE packet over the network
- * protocol connection ${NC} asking to read the specified file, which should
- * be ${size} (<= 262144) bytes long if ${size} is not (uint32_t)(-1).
+ * Construct and send a NETPACKET_READ_FILE packet asking to read the
```

Comment:
None of the functions in `netpacket.h` mention "the network protocol connection", so I figured that was a typo in `netpacket_read.c`.